src/main/java/frc/robot/constants/subsystems/DrivetrainConstants.java

```diff
@@ -252,6 +252,9 @@ public class DrivetrainConstants {
   public final Double kPositionTolerance = 0.25;
   public final Double kVelocityTolerance = 0.1;
 
+  public final Double kDriveTranslationMaxVelocitySlow = 1.5;
+  public final Double kDriveTranslationMaxAccelerationSlow = 1.0;
```

Comment:
This seems too slow imo, we could probably drive at like 2 m/s and do 3 m/s^2. We can build up on the practice field though